doc/nrf/ug_matter_overview_dev_model.rst

```diff
@@ -0,0 +1,32 @@
+.. _ug_matter_overview_dev_model:
+
+Matter development model
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter stack is developed on the official `Matter GitHub repository`_ and is open-sourced.
+
+The Matter stack implementation contains separation between platform-agnostic and platform-dependent functionality.
+The open-source implementation offers ports for several resource-constrained, embedded SoCs as well as POSIX-based platforms.
+
+Matter strives for reusing technologies from market-proven solutions, such as Apple HomeKit or Google Weave.
+Wi-Fi and :ref:`ug_thread` are its main wireless connectivity protocols that offer seamless integration with other IPv6-based networks and are application-layer agnostic.
+Bluetooth® LE can be used for commissioning of the Matter accessories, and QR codes and NFC tags can be used to initiate the commissioning.
+
+For more information about Matter architecture, read :ref:`ug_matter_overview_architecture_integration`.
+
+Matter in the |NCS|
+*******************
+
+The |NCS| provides full toolchain for Linux, macOS, and Windows, and is built on top of the Zephyr RTOS.
+It includes west for managing repositories, toolchain manager for managing toolchain, Kconfig for feature configuration, and Devicetree for board description.
+Finally, it integrates the OpenThread and Bluetooth LE stacks, which can work in multiprotocol operation.
```

Comment:
I don't know how to write this sentence grammatically correct, but I think you got my point :) I want to mention also Wi-Fi, but do not suggest that Wi-Fi can work with Thread in a multiprotocol way.
```suggestion
Finally, it integrates the OpenThread and Wi-Fi stacks, which can work in multiprotocol operation with integrated Bluetooth LE stack.
```